File: checkYear.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void){

int givenMonth = get_int("What month from the past?");
int givenYear = get_int("What year from the past?");

int currentYear = get_int("What is the current year?");
int currentMonthNumber = get_int("What is the current month(number)?");
int monthDifference = currentMonthNumber - givenMonth;
int yearDifference = currentYear - givenYear;
printf( "This is the difference between the given date and the current date %i years and %i  months\n", abs(yearDifference), abs(monthDifference));
}

File: checkYear.js
var givenMonth = 12;
var givenYear = 1984;

var d = new Date();
var months = ["January","February","March","April","May","June","July","August","September","October","November","December"];

var todayMonth = months[d.getMonth()];
var currentYear= d.getFullYear();

var currentMonthNumber = months.indexOf(todayMonth) + 1 ;

console.log("This is the number of the current month: " + currentMonthNumber);

console.log( " This is the number of the current year: " + currentYear);


console.log( "This is the difference between the given month and the current month: " + (currentMonthNumber - givenMonth) );

var monthDifference = currentMonthNumber - givenMonth;



var yearDifference = currentYear - givenYear;
console.log( "This is the difference between the given year and the current year: " + yearDifference + " years and " + Math.abs(monthDifference) + " months");
